### PR TITLE
Emotion

### PR DESCRIPTION
**Pull request by Klaypexx**
## About the PR
Adds sound emotion of crying, whistle, squishes (only for slime). Also add salute emotion sound when we type o7 or salute in chat.

**Media**
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Sanchez_
- add: Added new emotion sound

### CODE REVIEW

File: Resources/Audio/Effects/Emotes/attributions.yml [hunk 1]
```diff
@@ -18,3 +18,8 @@
   license: "CC-BY-4.0"
   copyright: "Made by TV_LING. Converted from WAV to OGG and downmixed to 1 channel."
   source: "https://freesound.org/people/TV_LING/sounds/523467/"
+- files:
+    - salute.ogg
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from https://github.com/ss220-space/Paradise. No changes were made"
+  source: "https://github.com/ss220-space/Paradise/commit/89943a231ced9fc8db0aef65ff87093efee16b4e"
```
Comment (metalgearsloth) [inline Resources/Audio/Effects/Emotes/attributions.yml:25]: Dead links.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Audio/Effects/Emotes/attributions.yml (hunk 1 above)
Comment (Klaypexx) [reply Resources/Audio/Effects/Emotes/attributions.yml:25]: should I attach newer links for them?

**Comment by metalgearsloth** [comment]
Do you know where the original source of these are coz I've tried looking at paradise before and saw sussy licensing.

**Comment by Klaypexx** [comment]
> Do you know where the original source of these are coz I've tried looking at paradise before and saw sussy licensing (specifically the salute with the original source uncredited).

Yes, I see, I also tried to find the original source, including at https://github.com/ParadiseSS13 but they don’t have a whistle.ogg file or the like. I think it's better to find a similar sound to freesound with CC-BY license

**Comment by Klaypexx** [comment]
> SGTM just give me a buzz when it's done.
> 
> Also use "Custom" for the one license that doesn't conform to the others and specify it in the copyright.

After looking through a bunch of other sites on the Internet I did not find anything similar to this sound. Perhaps it was made by the contributor of this PR himself. If so, it is probably covered by the same CC-BY-SA-3.0 rights.

**Comment by Klaypexx** [comment]
> SGTM just give me a buzz when it's done.
> 
> Also use "Custom" for the one license that doesn't conform to the others and specify it in the copyright.

Found the original source of this sound from Baystation12

File: Resources/Audio/Effects/Emotes/attributions.yml (hunk 1 above)
Comment (mirrorcult) [reply Resources/Audio/Effects/Emotes/attributions.yml]: second link isnt dead, first link is just embedding badly with the period at the end, so just remove it

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**